runtime/src/main/java/org/corfudb/runtime/view/LayoutView.java

```diff
@@ -164,10 +164,6 @@ public Layout prepare(long epoch, long rank)
                     .filter(x -> x != null)
                     .toArray(LayoutPrepareResponse[]::new);
 
-            log.debug("prepare: Successful responses={}, needed={}, timeouts={}, "
-                            + "wrongEpochRejected={}",
-                    acceptList.length, getQuorumNumber(), timeouts, wrongEpochRejected);
-
```

Comment:
Instead of logging this on each iteration, or removing the logs, you can just move it outside the loop. These logs show how long each phase takes which is useful when debugging.

Reply:
1. I don't know anyone who used it at least ones when debugging
2. We have all necessary logs on the server-side
Mostly those messages are minor. 

Anyway, we can move it. Will do

Reply:
Again these logs printing tens of thousands of times to the log ( for the last 3 years) and no one paid attention and no one fixed it, which means the messages were  always ignored when debugging

Reply:
@xnull Actually I am trying to fix it if you remember... and I did meet a problem that cluster kept bumping epoch from 0 to 300+ in a short time. For that issue, I don't know what happened because those counters are inaccurate.